llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -10,115 +10,129 @@
 // RISC-V subtarget features and instruction predicates.
 //===----------------------------------------------------------------------===//
 
+class RISCVExtension<string n, int major, int minor, string f, string v,
+                     string d, list<RISCVExtension> i = []>
```

Comment:
Does `i` mean `implies`? Can we change it to an accurate name?

Reply:
And other single-letter variables? Though they come from Target.td, but I think we should make it verbose so that they are friendly to new comers.